README.md

```diff
@@ -8,99 +8,166 @@ _user_ not a _developer_.
 moment, waiting in traffic, they are a _user_ of the car. But then they take a
-right turn and *cough*, *splutter* their car chokes to a stop. At once their
-mind changes to being a _mechanic_ or a _developer_. They think about
+right turn and *cough*, *splutter* &mdash; their car chokes to a stop. At once
+their mind changes to being a _mechanic_ or a _developer_. They think about
 petroleum, spark-plugs and ignitions. They can _debug_ their car using
 techniques they've learned. They can imagine what might be wrong. To get more
 data or to verify their guess, they _open up the hood_. Learning to be a
 developer (_computer mechanic_?) is similar. You must start "looking under the
 hood" of your computer. To access the "hood's" contents, you must use the
-shell.
+terminal.
 
-Developers "talk" to the computer by engaging it at the shell level. The shell
+Developers "talk" to their computers through the terminal. The terminal
 is just another way to collaborate with the computer. It might feel
 "old-school" or clunky at first compared with Windows or MacOSX. As you learn
 more, though, it might surprise you!
 
-Shell skills are critical in being a developer or HTML author.  Comfort in the
-shell is an assumption for many technical jobs.
+Terminal skills are critical in being a developer or HTML author.  Comfort in the
+terminal is an assumption for many technical jobs.
 
 ## Learning Objectives
 
-1. Define what a shell program is
+1. Define what a terminal program is
 2. Recognize a shell interface
-3. Recognize operating systems providing a shell
+3. Recognize operating systems providing a terminal
 
-## Define What a Shell Program Is
+## Define What a Terminal Program Is
 
-The shell is the program on your computer that takes in commands you type and
-takes action based on those commands. You can think of yourself as having a
-conversation with the computer.
+### Advice
 
-When you see a shell it will be at a "command prompt." This means, from the
-computer's perspective "I'm listening..." Prompts may look different from
-computer to computer. Customized prompts express developer uniqueness. We've
-seen weather reports, beautiful colors, computer temperatures in prompts. At
-heart, though, they they all carry your command to the operating system and run
-it.
+First of all, we have to be honest here: the difference between "shell" and
+"terminal" is quite easy to get confused by. To make things worse, developers
+are often imprecise and say one to mean the other and vice versa.
 
-You can ask the computer to do many types of tasks. From the command prompt you
-can move or rename files just like you would do from a window environment.  You
+In order to try to help build your instinct for understanding the concepts,
+we'll often "over-explain" or make a sentence seem ridiculously long. We're
+doing this intentionally in order to try to help you build an intuition about
+the actors in this story.
```

Comment:
I really like the explanation of teaching concepts here, but it's starting to get too smart again. Maybe instead of "We're doing this intentionally in order to try to help you build an intuition about the actors in this story.", something like "We're doing this intentionally in order to try to help you build your developer intuition".